bindata/oauth-openshift/deployment.yaml

```diff
@@ -102,6 +102,9 @@ spec:
             - name: v4-0-config-system-trusted-ca-bundle
               readOnly: true
               mountPath: /var/config/system/configmaps/v4-0-config-system-trusted-ca-bundle
+            - name: oauth-server-audit
```

Comment:
see if https://github.com/openshift/cluster-authentication-operator/blob/master/bindata/oauth-apiserver/deploy.yaml#L36-L51 is needed here, too